### ISSUE TITLE
Masters report REGIST status

### ISSUE THREAD

**Issue by tackong** [naver/searchad-apidoc#258]
Hello, could you give more details about "REGIST" status in /masters-report response? Thanks!

**Comment by naver-searchad** [comment]
Hello, this is Naver Searchad API!

these are the status specifications
- REGIST: request registered
- RUNNING: it's running
- BUILT: request done with data existing
- NONE: request done with none data
- ERROR: error occurred

**Comment by tackong** [comment]
Hello, thanks for the reply! I'm recently trying to pull down master reports for all of our Naver accounts. Reports of some of the accounts could be fetched and parsed correctly, while there's one account's report always showing status as "REGIST" and never get into "RUNNING" or "BUILT", do you know what could happen there? Thank you!

**Comment by naver-searchad** [comment]
@tackong 
if you may let me know the account id that is always showing 'REGIST' status report, then faster we could reply to you.

**Comment by sem-naver** [comment]
@naver-searchad Hi, I'm a PM working along with @tackong for getting the master report. Since Account ID is a very private information, we can't share it in github. Is there another method we could share account ID more privately? Thank you!

**Comment by naver-searchad** [comment]
@tackong @sem-naver 
the job id will do.

**Comment by tackong** [comment]
I don't think report Id is OK to be shared in public either, do you have a private way like email so that we can send you the details? Thank you!

**Comment by naver-searchad** [comment]
@tackong 
As for resource ids like report, campaign etc are permitted to access only under authorization.
and resource id itself has no meaning.
You can guess other user's resource ids by changing some numbers on it and might test if you could get resource information by API.

But if you're still concerned, you can use 'help channel'
https://saedu.naver.com/support/inquiry.nhn

**Comment by tackong** [comment]
Hello, so we just ran a master report over out adgroup and the report still got stuck on "REGIST" status, the report Id is 807657EE2ED7B6AAFD25A00A172E4BDF, can you help with that?

**Comment by naver-searchad** [comment]
@tackong 
As for the report id you requested, below is the current response status
you can see the 'BUILT' status.
I hope this could be helpful for you.
```
{
  "id" : "807657EE2ED7B6AAFD25A00A172E4BDF",
  "fromTime" : "",
  "item" : "Adgroup",
  "downloadUrl" : "/report-download?authtoken=DSLSvNwOoSKiPyFrLiVHgFHnnGryGuf22gcIMY9Aa0WXss6PeI7U3XwifbljhLGxPE2N8GU%2Fw5SmUL6sFA0L5cy7x%2F%2B5qJuSkUrEFHdgegI%3D",
  "updateTime" : "2019-08-08T17:26:00Z",
  "status" : "BUILT",
  "managerLoginId" : "tripadvisor",
  "managerCustomerId" : 4XXXXX,
  "clientCustomerId" : 4XXXXX,
  "registTime" : "2019-08-08T17:27:08.918Z"
}
```

**Comment by tackong** [comment]
Thanks, does that mean the report for this adgroup won't be available immediately and we have to wait a random time before it's available? We actually would expect a quicker "REGIST" and "RUNNING" process for the report so the program doesn't have to wait endlessly or it will be great if we can know the estimated waiting time on the "REGIST" stage so we can have a separate job to pull the report when it's available.

**Comment by naver-searchad** [comment]
@tackong 
Normally, a job come down to 'BUILT' status in some seconds.
But is not guaranteed always.
We hope the 'seconds' is enough time for your another job to pull the status.
No need to wait 'endlessly'!

**Comment by tackong** [comment]
Hi Naver, so from what I saw in the test, all master groups of our 3 accounts trying to pull adgroup structures didn't finish after being stuck on "REGIST" for 15 minutes. I manually aborted the program since I'm not sure how much longer would I need to wait. This is a serious issue that we need to take care since it's blocking our sync process. We would really appreciate if there's a way to find out the estimated waiting time on "REGIST" so that we can schedule a separate job fetching the report when it's ready instead of waiting. Thank you!
